### PR TITLE
Dr/no apps fix

### PR DESCRIPTION
**Pull request by diegoribero**
- Cleaner exit
- Was crashing when user had no apps created

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -161,15 +161,18 @@ var msg = [];
     },
     function firebaseInit(cb) {
       debug('Starting Firebase...'.green + ' U:', Matrix.userId, ', D: ', Matrix.deviceId, ', DT: ' , Matrix.deviceToken);
-
+      Matrix.service.firebase.initialized = true;
```
Comment (eighteyes) [inline index.js]: This is false. No?

File: lib/service/index.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ var debug = debugLog('matrix');
     return (f[0] === '.' || f.indexOf('.swp') > -1 )
   })
 
-  files.forEach(function(file) {
-    f[file.slice(0,-3)] = require('./' + file);
-    debug('service ready:'.blue, file.slice(0,-3) );
+  files.forEach(function (file) {
```
Comment (eighteyes) [inline lib/service/index.js]: good catch, this keeps it consstent

**Comment by eighteyes** [comment]
Move the initialize = true into the init callback and this is good.

File: index.js (hunk 1 above)
Comment (diegoribero) [inline index.js]: You are most certainly right, fixed and pushed! :)

**Pull request merged**